### ISSUE TITLE
Battery Routing

### ISSUE THREAD

**Issue by Zylanx** [hazels-garage/satpad#2]
The battery should be routed to GND and RAW, not to headers in the position of B+ and B- on the nice!nano which are intended for direct connection of the battery.

This is the very first point on the official docs here: https://nicekeyboards.com/docs/nice-nano/getting-started/

"Do not install sockets or post headers to the B+ or B- pins (top pin on each side)
If you need to use these pins with your PCB, RAW and GND are the respective equivalents to B+ and B-"

**Comment by Zylanx** [comment]
I'd probably make a pull request but I broke my arm, unfortunately

**Comment by jasonhazel** [comment]
Yeah, you're 100% right on that. SatPad was my very first design and I definitely made some mistakes while making it. Nice!Nano support was an afterthought.

 I've moved on from this project though so won't be making any updates. I will accept pull requests.

**Comment by Zylanx** [comment]
All good! You're able to use kicad properly which is better than me! Once I'm back in action I'll remember to have a go
